src/platform/silabs/BLEManagerImpl.h

```diff
@@ -30,6 +30,8 @@
 #define BLE_MAX_CONNECTION_INTERVAL_MS 45 // 45 msec
 #define BLE_SLAVE_LATENCY_MS 0
 #define BLE_TIMEOUT_MS 400
+#define BLE_DEFAULT_TIMER_PERIOD_MS (1)
+#define BLE_SEND_INDICATION_TIMER_PERIOD_MS (5000) // Time kept to support all WiFi chips BLE (RS9116/ SiWx917 NCP/SOC)
```

Comment:
Since this is only used in the BleManagerImpl of the rs911x it can be in the .cpp file and not in the header.